src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,24 +1,25 @@
-Microsoft.CodeAnalysis.GeneratorAttribute.GeneratorAttribute(string firstLanguage, params string[] additionalLanguages) -> void
-Microsoft.CodeAnalysis.GeneratorAttribute.Languages.get -> string[]
-Microsoft.CodeAnalysis.GeneratorExecutionContext.SyntaxContextReceiver.get -> Microsoft.CodeAnalysis.ISyntaxContextReceiver
-Microsoft.CodeAnalysis.GeneratorInitializationContext.RegisterForSyntaxNotifications(Microsoft.CodeAnalysis.SyntaxContextReceiverCreator receiverCreator) -> void
+#nullable enable
```

Comment:
nit: technically `#nullable enable` is only needed in one of the files. I'd put it just in the shipped file.